lib/shared-ini/index.d.ts

```diff
@@ -15,5 +15,16 @@ export interface IniFileContent {
  */
 export function loadFrom(options: LoadFileOptions): IniFileContent;
 
+/**
+ * Read specified file and return parsed config object. This method will always
+ * read from disk and won't update cache. This is a lower level function of 
+ * loadFrom().
+ * @param filename [string] valid readable file path containing aws credentials
+ * or aws configs
+ * @param isConfig [boolean] true if specified file is an aws config file; false
+ * if the file is an aws credentials file
+ */
+export function parseFile(filename: string, isConfig: boolean): IniFileContent;
```

Comment:
Are the typings in this file accurate? It looks like the public interface we are exposing is the `IniLoader` class, but these are just functions (with no clear way on how to import them as a consumer of our library)

Reply:
I put them this way to get around named exporting. So what I'd like to do is `require('shared-ini').loadFrom()` in commonJS or `import {loadFrom} from 'shared-ini'` in esm. But in type definition I cannot do default export(`export default`) of a singleton instance. So I have to explicitely export the public properties of the instance I'd like to export.